apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -253,6 +263,29 @@ router.post(
         }
       });
       return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
+    } else if (req.body.__action === 'send_grades') {
+      // Should something like this be in a model?
+      const assessment = await queryRow(
+        sql.select_assessment_with_course_instance,
```

Comment:
`_with_course_instance` typically indicates that this would return a `course_instance`, but it's just checking membership. I'd call this `select_assessment_in_course_instance` instead.